nitro-protocol/test/contracts/ForceMove/challenge.test.ts

```diff
@@ -235,15 +246,22 @@ describe('challenge', () => {
         expect(eventFixedPart[2]).toEqual(fixedPart.channelNonce);
         expect(eventFixedPart[3]).toEqual(fixedPart.appDefinition);
         expect(eventFixedPart[4]).toEqual(fixedPart.challengeDuration);
+
         expect(eventIsFinal).toEqual(isFinalCount > 0);
-        expect(
-          parseOutcomeEventResult(
-            eventSignedVariableParts[eventSignedVariableParts.length - 1].variablePart.outcome
-          )
-        ).toEqual(signedVariableParts[signedVariableParts.length - 1].variablePart.outcome);
-        expect(
-          eventSignedVariableParts[eventSignedVariableParts.length - 1].variablePart.appData
-        ).toEqual(signedVariableParts[signedVariableParts.length - 1].variablePart.appData);
+
+        if (proof.length > 0) {
```

Comment:
Should we add some expectations for the candidate part (which is never empty?)

Reply:
It is right below, see [261-264](https://github.com/statechannels/go-nitro/pull/817/files/c2c0264c775401a3b1edac3bc49c03c92c3edaf7..bf8e9e33e346e587597721ee306955a9af086a49#r950047050).

Reply:
Sure it is! Not sure how I missed it!